Basic/Pod/Dataflow.pod

```diff
@@ -173,6 +173,11 @@ All slices or transformations that simply select a subset of the original
 pdl are two-way. Matrix inverse should be. No arithmetic
 operators are.
 
+A note on (native) complex numbers: At the moment, creal, etc. are, in effect, slicing
+operations and not projections with type casts. This means, dataflow is
+active. This is still under discussion and the behaviour may change. Remember
```

Comment:
This is currently the opposite of true. `creal` et al just copy out the real or imaginary component.